src/main/java/gift/model/Gift.java

```diff
@@ -1,20 +1,32 @@
 package gift.model;
 
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "gift")
 public class Gift {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+    @Column(name = "name")
+    @NotNull
     private String name;
+    @Column(name = "price")
+    @NotNull
     private int price;
+    @Column(name = "imageUrl")
+    @NotNull
     private String imageUrl;
 
     public Gift() {
     }
 
-    public Gift(Long id, String name, int price, String imageUrl) {
+    public Gift( String name, int price, String imageUrl) {
```

Comment:
```suggestion
    public Gift(String name, int price, String imageUrl) {
```

참고로 Command + Alt + L (맥북이면 Command + Option + L) 누르면 자동 줄정렬이 가능합니다 ㅎㅎ

Reply:
습관적으로 자동 줄정렬을 해줘야하는데 자꾸 놓치게되네요 ㅜ 계속 눌러주는 습관을 들여보겠습니다.